.travis.yml

```diff
@@ -1,58 +1,72 @@
 language: node_js
 
+android:
+  components:
+    # Tools listed twice because doc says it's needed to get updates
+    # https://docs.travis-ci.com/user/languages/android/
+    - tools
+    - platform-tools
+    - tools
+
+    # The BuildTools version used by your project
+    - build-tools-26.0.1
+
+    # The SDK version used to compile your project
+    - android-24
+
+    # Additional components
+    - extra-google-m2repository
+    - extra-android-m2repository
+
+    # Specify at least one system image,
+    # if you need to run emulator(s) during your tests
+    - sys-img-armeabi-v7a-android-24
+
 matrix:
     include:
         - os: linux
-          dist: precise
           language: android
           sudo: required
-          android:
-            components:
-              # Uncomment the lines below if you want to
-              # use the latest revision of Android SDK Tools
-              - platform-tools
-              - tools
-
-              # The BuildTools version used by your project
-              - build-tools-25.0.0
-
-              # The SDK version used to compile your project
-              - android-25
-
-              # Additional components
-              - extra-google-m2repository
-              - extra-android-m2repository
-
-              # Specify at least one system image,
-              # if you need to run emulator(s) during your tests
-              - sys-img-x86-android-24
-              - sys-img-armeabi-v7a-android-24
+          dist: trusty
+          jdk: oraclejdk8
+          services:
+            - docker
+          env:
+            # Note that on trusty the emulator defaults to qemu2 (at least for
```

Comment:
You're missing the closed parentheses.

Reply:
Fixed in 782291689407df019b9804f17a4e38a759b2274f